docs/metric.md

```diff
@@ -0,0 +1,128 @@
+## Metric
+
+* [Create new metric](#create)
+* [Apply metric](#apply)
+* [List metrics](#list)
+* [Delete metric](#delete)
+
+### Create
+
+* Creates a new metric
+* Only metric name is required. `system-name` can be override with optional parameter.
+* `service` positional argument is a service reference. It can be either service `id`, or service `system_name`. Toolbox will figure it out.
+* This is not idempotent command. If metric with the same name already exists, command will fail.
+* Create a `disabled` metric by `--disabled` flag. By default, it will be `enabled`.
+* Several other options can be set. Check `usage`
+
+```shell
+NAME
+    create - create metric
+
+USAGE
+    3scale metrics create [opts] <remote>
+    <service> <metric_name>
```

Comment:
I think it might be better to use `-` instead of `_` to describe positional parameter placeholders like other tools do like `oc`, and as we do with the flag names